GNUmakefile

```diff
@@ -23,7 +23,7 @@ APISERVER=maya-apiserver
 # Specify the date o build
 BUILD_DATE = $(shell date +'%Y%m%d%H%M%S')
 
-all: test
+all: test bin apiserver
```

Comment:
Can the target bin be renamed to mayactl or something more appropriate ?

Reply:
Naming it as `mayactl`